PeerioTechnologies/peerio-icebear
# fix(download): avoid renaming to the same name

tempPath can be equal to filePath and then rename fails on mobile.
Made this mistake during refactoring.

#### Relevant info and issue/PR links 
 
#### Testing instructions  

* Download a file and see if it's downloaded.
* Make sure caching files doesn't fail.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
